README.md

```diff
@@ -104,13 +128,21 @@ Runs the component documentation.<br /> Open
 
 ### Linking
 
-If you are working with any @shapeshiftoss dependencies use `yarn link`
-These next steps will show you how to link to the `chain-adapters` repo
-From your projects folder:
+The lib repository contains packages used in multiple ShapeShift repositories, and is often needed to run projects locally. 
+To use the latest version of lib when running the web app, it is necessary to use yarn link so that you can use local
+changes to lib in your local web repo.
```

Comment:
If you're developing locally in this `web` repository, and need to make changes affecting packages in `lib` or `unchained` (backend), use the following steps to link packages locally for developing. If your changes only touch `web` these steps are not necessary.